logical-comaprison-operators.js

```diff
@@ -48,32 +48,49 @@ const isHappy = false;
 // - Check if num is between 10 and 20 (inclusive) using the logical AND operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
 
+
 // - Check if num is either less than 5 or greater than 50 using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+let numlees = num < 5
+console.log(numlees);
 
 // - Check if str is either "apple" or "orange" using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+let stror = "apple"||"orange"
+console.log(stror);
 
 // - Check if isHappy value is true using the logical NOT operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+let isHappynot = !isHappy
+console.log(isHappynot);
 
 // - Check if num is even and greater than 10 using the logical AND operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+let numeven = num >= 10
```

Comment:
You forgot to check if the number is even, you can do so by writing the following:
```
console.log( num % 2 === 0 && num > 10 )
```